pom.xml

```diff
@@ -150,7 +150,7 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
       <version>${hadoop.version}</version>
-      <scope>provided</scope>
```

Comment:
why remove provided scope?

Reply:
Design time validation is failing and getting NoClassDefFoundError for Hadoop classes. 
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/net/ftp/FTPClient
        at io.cdap.plugin.batch.source.ftp.FTPFileSystemImpl.connect(FTPFileSystemImpl.java:112) ~[na:na]
        at io.cdap.plugin.batch.source.ftp.FTPFileSystemImpl.getFileStatus(FTPFileSystemImpl.java:384) ~[na:na]
        at io.cdap.plugin.batch.source.ftp.FTPConfig.validate(FTPConfig.java:184) ~[na:na]
        at io.cdap.plugin.format.plugin.AbstractFileSource.configurePipeline(AbstractFileSource.java:87) ~[na:na]

Reply:
I see. It would be better to add a direct dependency on commons-net for the FTPClient class than make hadoop-common compile scope. hadoop-common will pull in a whole bunch of other dependencies.

Reply:
done